firestore/src/common/compiler_info.cc

```diff
@@ -163,9 +163,7 @@ std::ostream& operator<<(std::ostream& os, LanguageVersion) {
 struct StandardLibraryVendor {};
 
 std::ostream& operator<<(std::ostream& os, StandardLibraryVendor) {
-#if defined(_STLPORT_VERSION)
```

Comment:
I'm a little torn about removing this -- I guess it's possible (even if very unlikely) that someone could compile the SDK from the source code and modify it to work on STLPort. Also, unlike other cases, this isn't really a workaround, so it doesn't add to our tech debt, IMO. Unless you feel strongly about this, I'm leaning towards leaving this in.

Reply:
It would be quite a bit of work, but not impossible. My main point is that if we're acknowledging that there is a possibility of the SDK running on platforms we're not aware of (outputting `unknown`), we might as well have a check for STLPort, just for completeness' sake. I don't see how it costs us anything in terms of maintenance.

Reply:
Bump.

Reply:
Done